### PR TITLE
Made header image responsive. Fixes React-Indy/website#5

### PR DESCRIPTION
**Pull request by pirate-matt**
Made header image responsive.
Updated README to include info about linking PRs to issues.

### CODE REVIEW

**Review by ejmudrak — DISMISSED** [review]
LGTM 👍

File: README.md [hunk 1]
```diff
@@ -10,15 +10,18 @@ The contributing process documentation is still in-progress. The numbered list b
...
 1. Create a new branch from `main`
-2. Make changes and commit them to your branch
-3. Create a PR with your branch against this repo's `main`
-4. Request review(s) of your PR
-5. Allow at least 24-48 hours for review
-6. Address any review comments
-7. Repeat steps 4-6 as needed
-8. Allow your PR to be approved & merged
-9. Celebrate! 🎉
+1. Make changes and commit them to your branch
```
Comment (ejmudrak) [inline README.md:15]: @pirate-matt do you have a MD formatter that is getting rid of the numbering of these lists? I feel like we probably want to keep the sequential numbering for DX

Comment (ashleemboyer) [reply README.md:15]: Plus one to this -- can we update these numbers to be explicit?

**Review by ashleemboyer — CHANGES_REQUESTED** [review]
Great catch and quick fix! 👏🏻

Just one change requested and I'll approve. 😊

File: README.md (hunk 1 above)
Comment (pirate-matt) [reply README.md]: 👍🏻 updated!

**Comment by pirate-matt** [comment]
Switched markdown ordered lists to explicit numbering!

Fun fact you can run prettier `npm run prettier:fix` to fix this for you, but only if you don't have a `1.` for every list item 🙈. I didn't find a way to force prettier to pick one over the other. In the future, we'll likely want to get a more prescriptive linter/prettier for markdown if we plan on using markdown files for events/page content.

**Review by ashleemboyer — APPROVED** [review]
@pirate-matt Appreciate the changes, apologies for the delay here! 🙏🏻

> In the future, we'll likely want to get a more prescriptive linter/prettier for markdown if we plan on using markdown files for events/page content.

I 100% agree here. Whatever we can do to make things smooth and easy!

**Pull request merged**